packages/near-fast-auth-signer/src/utils/multi-chain/relayer/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface FunctionCall {
```

Comment:
we have this type defined on `import { FunctionCall } from '@near-js/transactions'` lets reference from original if possible?

Reply:
The types are different

This is the definition of the FunctionCall on '@near-js/transactions'

`export declare class FunctionCall extends IAction {
    methodName: string;
    args: Uint8Array;
    gas: BN;
    deposit: BN;
}`

Reply:
hmm, I thought we are constructing FunctionCall to do the same job.

Reply:
Just curiosity, when you say its different, I did notice that its `method_name` is different from `methodName` and `args` are in string. At the end, are we doing the same thing as classic signDelegate from NAJ? If we are doing the same thing shouldn't we fix the case issue? Or have completely different name for these Types? (So that users won't get confused why `FunctionCall` from NAJ VS `FunctionCall` from fast-auth-signer?